docs/pbc/all/miscellaneous/202407.0/tutorials-and-howtos/add-audit-log-types.md

```diff
@@ -0,0 +1,165 @@
+     * @param \Generated\Shared\Transfer\AuditLoggerConfigCriteriaTransfer $auditLoggerConfigCriteriaTransfer
+     *
+     * @return bool
+     */
+    public function isApplicable(AuditLoggerConfigCriteriaTransfer $auditLoggerConfigCriteriaTransfer): bool
+    {
+        return $auditLoggerConfigCriteriaTransfer->getChannelName() === $this->getChannelName();
+    }
+
+    /**
+     * @return string
+     */
+    public function getChannelName(): string
+    {
+        return 'checkout';
+    }
+
+    /**
+     * @return list<\Spryker\Shared\Log\Dependency\Plugin\LogHandlerPluginInterface>
+     */
+    public function getHandlers(): array
+    {
+        return [
+            new AuditLogTagFilterBufferedStreamHandlerPlugin(),
+        ];
+    }
+
+    /**
+     * @return list<\Spryker\Shared\Log\Dependency\Plugin\LogProcessorPluginInterface>
+     */
+    public function getProcessors(): array
+    {
+        return [
+           new PsrLogMessageProcessorPlugin(),
+           new EnvironmentProcessorPlugin(),
+           new ServerProcessorPlugin(),
+           new AuditLogRequestProcessorPlugin(),
+           new ResponseProcessorPlugin(),
+           new AuditLogMetaDataProcessorPlugin(),
+       ];
+    }
+}
+```
+
+</details>
+
+
+2. Register the plugin in the config:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Pyz\Yves\Log\Plugin\Log\YvesCheckoutAuditLoggerConfigPlugin;
+
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_YVES] = [
+    //other plugins
+    YvesCheckoutAuditLoggerConfigPlugin::class,
+];
+```
+
+3. To add the `Checkout` type for other applications, introduce `AuditLoggerConfig` plugins following the example and register them in the config file:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Pyz\Yves\Log\Plugin\Log\YvesCheckoutAuditLoggerConfigPlugin;
+use Pyz\Glue\Log\Plugin\Log\GlueBackendCheckoutAuditLoggerConfigPlugin;
+use Pyz\Glue\Log\Plugin\Log\GlueCheckoutAuditLoggerConfigPlugin;
+use Pyz\Zed\Log\Communication\Plugin\Log\MerchantPortalCheckoutAuditLoggerConfigPlugin;
+use Pyz\Zed\Log\Communication\Plugin\Log\ZedCheckoutAuditLoggerConfigPlugin;
+
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_ZED] = [
+    //other plugins
+    ZedCheckoutAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_GLUE] = [
+    //other plugins
+    GlueCheckoutAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_GLUE_BACKEND] = [
+    //other plugins
+    GlueBackendCheckoutAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_MERCHANT_PORTAL] = [
+    //other plugins
+    MerchantPortalCheckoutAuditLoggerConfigPlugin::class,
+];
+```
+
+
+Now you can add audit logs with `Checkout` type across the application. Example:
+
+```php
+<?php
+
+use Generated\Shared\Transfer\AuditLoggerConfigCriteriaTransfer;
+use Spryker\Shared\Log\AuditLoggerTrait;
+
+class AnyCheckoutClassWhereAuditLoggingIsNeeded
```

Comment:
what does this mean?
I'd prefer to demonstrate this on a real class.

@dunqan This is a clear violation of the principles we ask customers to follow - do not extend Core classes on the project without necessity. 
And here we actually explaining that this is the way to go forward...